src/App.css

```diff
@@ -13,10 +13,12 @@
   margin: 0;
   padding: 0;
   box-sizing: border-box;
+  font-family: Verdana, Geneva, Tahoma, sans-serif;
 }
 
```

Comment:
Great job @Simpleshaikh1 🤝

- [ ] As given in the project requirements, we are supposed to adhere to the design given in order to create a replica of this [given design](https://github.com/microverseinc/curriculum-react-redux/blob/main/bookstore/images/bookstore.png).
- [ ] Kindly ensure the borders are removed from the `progress buttons`, and the `add book` button.
- [ ] Also ensure the font colors of the texts and the colors of the buttons on your design match the design provided.
- [ ] Kindly ensure the elements of your design are well aligned just like in the design provided.
![your-cms](https://github.com/Simpleshaikh1/Book-Store/assets/63917725/8bdedf60-f6bb-4aae-984c-6efb99c0e0f4)